src/libraries/sendtohelixhelp.proj

```diff
@@ -157,21 +148,36 @@
     <HelixPreCommand Include="printenv | grep COMPlus" />
   </ItemGroup>
 
-  <PropertyGroup>
-    <HelixPreCommands>@(HelixPreCommand)</HelixPreCommands>
-    <IncludeHelixCorrelationPayload Condition="'$(HelixCorrelationPayload)' != '' and '$(TargetOS)' != 'Browser'">true</IncludeHelixCorrelationPayload>
-  </PropertyGroup>
+    <!-- 
+      We are hosting the payloads for the WASM/browser on kestrel in the xharness process.
+      We also run some network tests to this server and so, we are running it on both HTTP and HTTPS.
+      For the HTTPS endpoint we need development SSL certificate.
+    -->
+  <PropertyGroup Condition="''$(TestPackages)' != 'true' and $(TargetsWindows)' != 'true' and '$(BrowserHost)' != 'windows'">
+    <!-- install SDK so that, we could use `dotnet dev-certs https`. -->
+    <IncludeDotNetCli>true</IncludeDotNetCli>
```

Comment:
Will `IncludeDotNetCli` be set for Windows as well?

Reply:
Maybe we can set this one in the bottom with `HelixPreCommands`?

Reply:
if powershell works for windows, we do not need full SDK.

Reply:
I know but we still need `IncludeDotNetCli`

Reply:
Then we don't need it at all?